lib/ansible/modules/packaging/language/pip.py

```diff
@@ -474,6 +475,10 @@ def main():
                 err += err_venv
                 if rc != 0:
                     _fail(module, cmd, out, err)
+            else:
```

Comment:
you should remove this part and setup `the venv_created = False` on the top (~ L443)